salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
@derBroBro Were you able to get @s0undt3ch the information he was looking for? We'd love to come to a resolution here and get this PR merged once we can all agree on the best approach.